app/views/catalog/record/_mods_bibliographic.html.erb

```diff
@@ -25,5 +25,7 @@
 <% end %>
 
 <% document.mods.location.each do |location| %>
-  <%= render ModsDisplay::FieldComponent.new(field: location, label_html_attributes: { class: 'col-md-3' }, value_html_attributes: { class: 'col-md-9' }) %>
+  <% unless location.values.detect { |loc| /href=.#{Settings.PURL_EMBED_RESOURCE}/.match?(loc) } %>
```

Comment:
Can this condition get pulled up to `document.mods.location.reject { ... }.each |location|`?

Reply:
good call, will do